CondFormats/GeometryObjects/interface/PDetGeomDesc.h

```diff
@@ -2,25 +2,51 @@
 #define CondFormats_PDetGeomDesc_h
 
 #include "CondFormats/Serialization/interface/Serializable.h"
+#include "Geometry/VeryForwardGeometryBuilder/interface/DetGeomDesc.h"
+#include "DetectorDescription/DDCMS/interface/DDTranslation.h"
+#include "DetectorDescription/DDCMS/interface/DDRotationMatrix.h"
 
 #include <vector>
 #include <string>
 
 class PDetGeomDesc {
 public:
-  PDetGeomDesc(){};
-  ~PDetGeomDesc(){};
-
   struct Item {
+    Item() = default;
+    Item(const DetGeomDesc* const geoInfo) {
+      dx_ = geoInfo->translation().X();
+      dy_ = geoInfo->translation().Y();
+      dz_ = geoInfo->translation().Z();
+
+      const DDRotationMatrix& rot = geoInfo->rotation();
+      double xx, xy, xz, yx, yy, yz, zx, zy, zz;
+      rot.GetComponents(xx, xy, xz, yx, yy, yz, zx, zy, zz);
```

Comment:
Hi @ghugo83, I don't see `xx, xy, xz...` used anywhere else. Is it possible to feed `axx_, axy_, ...` directly into `GetComponents`?

Reply:
Yes, this code was moved out of PPSGeoemtryBuilder and kept identical. Will fix this though.